source/hardware/controllers/pioneer_ddj_sb3.rst

```diff
@@ -0,0 +1,381 @@
+Pioneer DDJ-SB3
+===============
+
+.. sectionauthor::
+   Javier Vilalta <javier at dancephy.com>
+
+.. figure:: ../../_static/controllers/pioneer_ddj_sb3.svg
+   :align: center
+   :width: 100%
+   :figwidth: 100%
+   :alt: Pioneer DDJ-SB3 (schematic view)
+   :figclass: pretty-figures
+
+   Pioneer DDJ-SB3 (schematic view)
+
+The Pioneer DDJ-SB3 is a 4 deck USB DJ controller.
+
+-  `Manufacturer's Product Page <https://www.pioneerdj.com/en-us/product/controller/ddj-sb3/black/overview/>`__
+-  `Manufacturer's User Manual <https://docs.pioneerdj.com/Manuals/DDJ_SB3_DRI1533A_manual/>`__
+-  `Manufacturer's Firmware Update <https://www.pioneerdj.com/en-us/product/controller/ddj-sb3/black/support/>`__
+-  `Mixxx User Forum <https://mixxx.discourse.group/t/ddj-sb3/18346>`__
+
+.. versionadded:: 2.3.0
+
+Audio Setup
+-----------
+
+The mapping relies on the following channel assignments:
+
+===================== ================
+Output Channels       Assigned to
+===================== ================
+1-2                   Master
+3-4                   Headphones
+===================== ================
+
+This controller mapping does not support any inputs. The controller itself has a microphone input, but that cannot be controlled via MIDI.
+
+Controller Mapping
+------------------
+
+A schematic drawing with the control numbers that are used here can be found on the specified page in the User Manual in the Links section.
+
+.. _pioneer-ddj-sb3-decks:
+
+Deck section (p. 6)
+~~~~~~~~~~~~~~~~~~~
+
+.. figure:: ../../_static/controllers/pioneer_ddj_sb3_decks.svg
+   :align: center
+   :width: 65%
+   :figwidth: 100%
+   :alt: Pioneer DDJ-SB3 (deck section)
+   :figclass: pretty-figures
+
+   Deck section (p. 6)
+
+===  =================================================  ============================================================================================
+No.  Control                                            Function
+===  =================================================  ============================================================================================
+1    :hwlabel:`TEMPO` slider                            Adjust track playing speed (can be adjusted via :hwlabel:`SHIFT` + :hwlabel:`BEAT SYNC`)
+2    Jog Wheel (top)                                    Pitch bend (nudge)
+2    :hwlabel:`VINYL` + Jog Wheel (top)                 Scratch (move play position)
```

Comment:
```suggestion
2    Jog Wheel (top)                                    Pitch bend (nudge) or scratch when vinyl mode is enabled
```

The way this is written had me concerned that scratching only worked with the vinyl button held down.